### PR TITLE
Fix Gitter bridge feature list formatting (again)

### PR DESCRIPTION
**Pull request by MadLittleMods**
Fix Gitter bridge feature list formatting: https://matrix.org/bridges/#gitter

Part of https://github.com/matrix-org/matrix.org/issues/903

Follow-up to https://github.com/matrix-org/matrix.org/pull/919 and https://github.com/matrix-org/matrix.org/pull/922

### CODE REVIEW

File: gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx [hunk 1]
```diff
@@ -22,15 +22,15 @@ If you're already using [Element](https://element.io), you can browse all of the
...
+✓ Native feeling usernames/avatar on both sides (virtual users)  
+✓ Replies/Threaded conversations  
+✓ Formatted Messages  
+✓ Mentions  
+✓ Edits  
+✓ Redactions/Deletions  
+✓ Status `/me` messages  
 ✓ Media/Files  
-✗ Reactions
+✗ Reactions
```
Comment (MadLittleMods) [inline gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:33]: I guess the key is double spaces after everything 🤷

Comment (benparsons) [reply gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:33]: it is, markdown, sorry I didn't check on this!

**Review by benparsons — APPROVED** [review]
lgtm!

File: gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx (hunk 1 above)
Comment (MadLittleMods) [reply gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx:33]: Ahh didn't know this trick. Thanks for the markdown hint!

> When you do want to insert a <br /> break tag using Markdown, you
end a line with two or more spaces, then type return.
>
> https://daringfireball.net/projects/markdown/syntax#p

**Pull request merged**